code/game/objects/items/cards_ids.dm

```diff
@@ -383,6 +383,61 @@
 		user.visible_message(span_notice("[user] shows you: [icon2html(src, viewers(user))] [src.name][minor]."), span_notice("You show \the [src.name][minor]."))
 	add_fingerprint(user)
 
+/obj/item/card/id/attack_hand_secondary(mob/user, list/modifiers)
+	. = ..()
+	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
+		return
+	/// Right clicking will deactivate any current holopays
+	if(my_store)
+		QDEL_NULL(my_store)
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+	if(!registered_account || !registered_account.account_job)
+		balloon_alert(user, "no account")
+		to_chat(user, span_warning("You need a valid bank account to do this."))
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+	/// Determines where the holopay will be placed based on tile contents
+	var/turf/projection
+	var/turf/step_ahead = get_step(user, user.dir)
+	var/turf/user_loc = user.loc
+	if(can_proj_holopay(step_ahead))
+		projection = step_ahead
+	else if(can_proj_holopay(user_loc))
+		projection = user_loc
+	if(!projection)
+		balloon_alert(user, "no space")
+		to_chat(user, span_warning("You need to be standing on or near an open tile to do this."))
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+	/// Valid tile for holopay placement
+	my_store = new(projection, user, src, registered_account)
```

Comment:
Should give this a slight delay so it's not too much of a spam tool. May also need a way for others to get rid of it